system/shaders/GL/1.2/gl_yuv2rgb_basic.glsl

```diff
@@ -122,13 +122,9 @@ vec4 process()
 #endif
 
 #if defined(XBMC_COL_CONVERSION)
-  rgb.r = pow(rgb.r, m_gammaSrc);
-  rgb.g = pow(rgb.g, m_gammaSrc);
-  rgb.b = pow(rgb.b, m_gammaSrc);
+  rgb.rgb = pow(rgb.rgb, vec3(m_gammaSrc,m_gammaSrc,m_gammaSrc));
```

Comment:
`vec3(m_gammaSrc)` is enough as I know GLSL
the same below

Reply:
I tried but it did not compile